frameworklauncher/src/main/java/com/microsoft/frameworklauncher/webserver/LauncherModule.java

```diff
@@ -301,48 +324,41 @@ public Response deleteMigrateTask(
   @GET
   @Path(WebStructure.FRAMEWORK_PATH)
   @Produces({MediaType.APPLICATION_JSON})
-  public AggregatedFrameworkStatus getFramework(
+  public FrameworkInfo getFramework(
```

Comment:
>FrameworkInfo [](start = 9, length = 13)

Maybe it's better to return the SummarizedFrameworkInfo here to make it consistent with framework listing API?

Reply:
FrameworkInfo is also consistent with framework listing API, but provide all infos for the framework, user should like it,

---
In reply to: [166511071](https://github.com/Microsoft/pai/pull/309#discussion_r166511071) [](ancestors = 166511071)